### PR TITLE
Update README.adoc

### PR DESCRIPTION
**Pull request by gkwan-ibm**
#24

### CODE REVIEW

**Comment by gkwan-ibm** [comment]
hi @Charlotte-Holt would you review the change?

File: README.adoc [hunk 1]
```diff
@@ -321,6 +321,6 @@ through the `MP_MESSAGING_INCOMING_PROPERTYREQUEST_GROUP_ID` environment variabl
 
 == Great work! You're done!
 
-You just developed a reactive Java application using MicroProfile Reactive Messaging, Open Liberty, and Kafka.
+You have successfully integrated a RESTful microservice with a reactive system by using MicroProfile Reactive Messaging.
```
Comment (Charlotte-Holt) [inline README.adoc:324]: @gkwan-ibm I'd just take out "have" so it's:

>You successfully integrated a RESTful microservice with a reactive system by using MicroProfile Reactive Messaging.

Comment (gkwan-ibm) [reply README.adoc]: @Charlotte-Holt I updated.

Comment (Charlotte-Holt) [reply README.adoc]: @gkwan-ibm Thanks! Approved.

**Pull request merged**